transmit-lib/src/cmr/transmit/indexer.clj

```diff
@@ -1,10 +1,77 @@
 (ns cmr.transmit.indexer
   "Provides functions for accessing the indexer application"
-  (:require [cmr.transmit.http-helper :as h]
-            [cmr.transmit.config :as config]
-            [cmr.transmit.connection :as transmit-conn]))
+  (:require
+   [cheshire.core :as cheshire]
+   [clj-http.client :as client]
+   [cmr.common.services.errors :as errors]
+   [cmr.transmit.http-helper :as h]
+   [cmr.transmit.config :as config]
+   [cmr.transmit.connection :as conn]))
 
 ;; Defines health check function
 (h/defhealther get-indexer-health :indexer {:timeout-secs 2})
 
 (h/defcacheclearer clear-cache :indexer)
+
+(defn- rebalance-collection-url
+  [conn index-set-id concept-id]
+  (format "%s/index-sets/%s/rebalancing-collections/%s" (conn/root-url conn) index-set-id concept-id))
+
+(defn- start-rebalance-collection-url
+  [conn index-set-id concept-id]
+  (str (rebalance-collection-url conn index-set-id concept-id) "/start"))
+
+(defn- finalize-rebalance-collection-url
+  [conn index-set-id concept-id]
+  (str (rebalance-collection-url conn index-set-id concept-id) "/finalize"))
+
+(defn get-index-set
+  "Submit a request to index-set app to fetch an index-set assoc with an id"
```

Comment:
There is no index-set app anymore :)

Reply:
was a lot easier to global replace cubby to redis :(. Fixed.